### DISCUSSION TITLE
Magick.pm

### DISCUSSION THREAD

**Discussion by mmfarmani** [ImageMagick/ImageMagick#2067] [Help]
| use Image::Magick;
  | my $image = Image::Magick->new();
  | $image->BlobToImage(substr($read_data, $pos2, $size));
  | # $image->Scale('640');
  | $image->Write("$upload_dir/$name");
  | }

The above code, in index.cgi , returns an error message : " Need to upload Magick.pm to /bin folder...

What does it mean? Where should I get Magick.pm ?

Thanks

**Comment by fmw42** [comment]
That is the Perl module. See https://github.com/ImageMagick/ImageMagick6/blob/master/PerlMagick/Magick.pm

**Reply by mmfarmani** [reply]
Thanks,
How can I download the file ?
I need to upload it to /bin folder of my site.
